packages/atlas/src/chat.ts

```diff
@@ -41,8 +41,8 @@ export class Chat extends Base {
     return await this.get({ route: `/api/v1/chat/room/members/${room}` });
   }
 
-  async findCallRoom(call: number): Promise<IRoom.FindCallRoomApiResponse> {
-    return await this.get({ route: `/api/v1/chat/room/call/${call}` });
+  async findSessionRoom(session: string): Promise<IRoom.FindSessionRoomApiResponse> {
+    return await this.get({ route: `/api/v1/chat/room/session/${session}` });
   }
```

Comment:
We can remove this api call as we don't need it in the frontend. Incase it is in-use, you can disable the frontend logic.